hack/run-e2e.sh

```diff
@@ -83,8 +86,10 @@ function e2e::image_load() {
         # pingcap/coredns:latest and nginx:latest is required for test
         # we suppose that you could pull this image on your host docker
         echo "info: load images pingcap/coredns:latest and nginx:latest"
-        docker pull pingcap/coredns:latest
-        docker pull nginx:latest
+        docker pull ${REGISTRY_MIRROR}pingcap/coredns:latest
+        docker pull ${REGISTRY_MIRROR}nginx:latest
```

Comment:
The `registry_mirror` of the base image has been set. I don't think it would be a problem to pull these images (if the mirror works well).

Reply:
just a try, seems it doesn't work